src/components/pages/Analytics/TechStackGraph.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import axios from 'axios';
+import embed from 'vega-embed';
+
+export default function TechStackGraph() {
+  let url = `${process.env.REACT_APP_DS_API_URL}/graph/tech-stack-by-role`;
+
+  axios
+    .get(url)
+    .then(res => {
+      embed('#vis', url);
```

Comment:
Is this line correct? Is the `embed` function supposed to make use of the `res` variable? If not, what's the point of the axios call?

Reply:
Good catch Mike! We have implemented a change that removes axios and places embed inside a useEffect hook.